component/otelcol/receiver/opencensus/opencensus.go

```diff
@@ -28,7 +28,7 @@ func init() {
 type Arguments struct {
 	CorsAllowedOrigins []string `river:"cors_allowed_origins,attr,optional"`
 
-	GRPC *GRPCServerArguments `river:"grpc,block,optional"`
+	GRPC *GRPCServerArguments `river:",squash"`
```

Comment:
Yep; we need to apply defaults for the arguments instead of the block in isolation. (Technically what it does now is incorrect, and would fail if the grpc block isn't provided)

Reply:
Basically this needs to be changed to align with what the zipkin receiver does for default arguments

Reply:
Thanks very much for spotting this. I tried to alight it with the zipkin receiver now.